Vungle/src/main/java/com/mopub/mobileads/VungleAdapterConfiguration.java

```diff
@@ -67,6 +72,7 @@ public void initializeNetwork(@NonNull final Context context, @Nullable final Ma
         boolean networkInitializationSucceeded = false;
         synchronized (VungleAdapterConfiguration.class) {
             try {
+                tokenReference.set(Vungle.availableBidTokens());
```

Comment:
This open API depends on not merged HBP epic PR: https://github.com/Vungle/vungle-android-sdk/pull/1076